manifests/server/install.pp

```diff
@@ -1,5 +1,7 @@
 # == Class dns::server
 #
+include dns::server::default
```

Comment:
instlal.pp:3 still needs to be removed. It is a include line outside a class definition and will be included by simply parsing this file, which is surprising and impossible to test.